yt/config.py

```diff
@@ -65,6 +79,19 @@
     ),
 )
 
+ytcfg_defaults["logging"] = (
+    dict(
+        level="INFO",
+        use_color=False,
+        stream="stderr",
+        handler="legacy",
+        format=r"%(message)s",
+        width=-1,  # <0 means "auto"
+        date_format="[%m/%d/%Y %H:%M:%S]",
+        custom_theme="",
+    ),
+)
```

Comment:
This should be a dict, not a tuple!

```suggestion
ytcfg_defaults["logging"] = dict(
    level="INFO",
    use_color=False,
    stream="stderr",
    handler="legacy",
    format=r"%(message)s",
    width=-1,  # <0 means "auto"
    date_format="[%m/%d/%Y %H:%M:%S]",
    custom_theme="",
)
```

Reply:
Thank you so much !